src/pages/SupplierInvoicePage.js

```diff
@@ -88,7 +88,7 @@ export class SupplierInvoicePage extends React.Component {
     database.write(() => {
       switch (key) {
         case 'numberOfPacks':
-          transactionBatch.numberOfPacks = parsePositiveFloat(newValue);
+          transactionBatch.setTotalQuantity(database, parsePositiveInteger(newValue));
```

Comment:
If a the pack size field is re-introduced, this will be a gotcha (because of `his.numberOfPacks = this.packSize ? quantity / this.packSize : 0;` in `setTotalQuantity`. Also if there is a value in this field and invalid value is entered, the field will be set to 0... 
Would rather a simple 
```
const numberOfPacks = parsePositiveInteger(newValue);
if (numberOfPacks !== null) transactionBatch.numberOfPacks = numerOfPacks;
```

Reply:
parsePositiveInteger will return 0 for all invalid inputs as of the next release of that module, so  that check will be redundant :). I can see that packsize may well sneak it's way back as a interface field, so sure.

Reply:
why would parsePositiveInteger return 0 ? This will override previous value of all inputs if invalid value is entered

Reply:
As per discussion in sussol-utils PR, sounds like parsePositiveInteger will be returning 0 even if value entered in invalid.. I am fine with this for now.

But can you make a comment in onEndEditing re the first part of my comment (i.e. if the pack size is re introduced we cannot use setTotalQuantity

Reply:
Just realised to change the switch to 'quantity' rather than 'numberOfPacks'. The column is called Quantity, so I think this is clearer as it is now. If that the column actually represents number of packs, the header should say that too to be really clear in my opinion. Adding comment as suggested too.